tests/unit/htdocs/modules/protector/Protector364RcFixesTest.php

```
@@ -0,0 +1,441 @@
+                    "Attacker-controlled request metadata value '{$value}' must not poison doubtfuls"
+                );
+            }
+        } finally {
+            $_SERVER                                   = $priorServer;
+            $_GET                                      = $priorGet;
+            $_POST                                     = $priorPost;
+            $_COOKIE                                   = $priorCookie;
+            $protector->_conf['dblayertrap_wo_server'] = $priorWoServer;
+            $protector->_dblayertrap_doubtfuls         = [];
+        }
+    }
```

PHPMD (2.15.0)

[error] fix15_behaviouralRequestMetadataDoesNotPoisonDoubtfuls accesses the super-global variable $_SERVER. (undefined)

(Superglobals)